### DISCUSSION TITLE
Practica CSS finalizada - Vacunacion

### DISCUSSION THREAD

**Discussion by Jramo5** [Launch-X-Latam/MisionFrontEnd#355] [Show and tell]
Hola amigos, aqui les comparto mi practica de CSS, la intente hacer lo mas parecida posible.
Aqui les dejo el link de la pagina ---> [https://jramo5.github.io/Launch-X-Latam-MisionFrontEnd/CSS/vacunacion/index.html](https://jramo5.github.io/Launch-X-Latam-MisionFrontEnd/CSS/vacunacion/index.html).

La hice lo mas responsiba posible para pantallas:

Móviles: + 320px
Tablet: + 720px
Portátil: + 1024px

Espero leer sus opiniones al respecto, gracias.

También les dejo los trabajos anteriores para que los vean ---> [https://jramo5.github.io/Launch-X-Latam-MisionFrontEnd/](https://jramo5.github.io/Launch-X-Latam-MisionFrontEnd/)

**Comment by DiegoCorchuelo** [comment]
Te quedo increíble, podrías explicarme como hiciste las media queries?

**Reply by Jramo5** [reply]
Hola, para serte sincero no tengo mucha practica con los medias query, si revisas mis estilos ---> [https://github.com/Jramo5/Launch-X-Latam-MisionFrontEnd/blob/main/CSS/vacunacion/style/media-query.css](https://github.com/Jramo5/Launch-X-Latam-MisionFrontEnd/blob/main/CSS/vacunacion/style/media-query.css) veras que tengo algunas "malas practicas".

El desarrollo del sitio lo comencé desde la versión móvil y va desde los 0px hasta los 767px, después fui reacomodando las cajas y los estilos según como deseaba.

Algun ejemplo seria asi:

@media screen and (min-width:768px){

      div.Caja-padre {
                   width: 300px;

}

}
Este ejemplo es una caja (div) que apartir de los +768px medira un ancho de 300px.

**Reply by DiegoCorchuelo** [reply]
Muchas gracias
